src/ckeditor.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* global CKEDITOR */
+
+import { getEditorNamespace } from './utils/geteditornamespace.js';
+
+export default {
+	name: 'ckeditor',
+
+	render( createElement ) {
+		return createElement( 'div', {}, [
+			createElement( this.tagName )
+		] );
+	},
+
+	props: {
+		value: {
+			type: String,
+			default: ''
+		},
+		type: {
+			type: String,
+			default: 'classic',
+			validator: type => [ 'classic', 'inline' ].includes( type )
```

Comment:
What about `divarea` editor? 🤔

Reply:
`divarea` is just a plugin, why it should be considered as a type?

Reply:
Yes, you are right that `divarea` is only the plugin and can be simply added so it makes sense. I'm not sure TBH for some reason we have `classic`, `divarea`, `inline` in Angular integration but only `classic` and `inline` in React.
I think in Angular, there was an issue with Classic one (iframed) and we went with divarea plugin to temporarily substitute it 🤔 

Looking at our docs, the type is Classic/Inline so we should stick with it to be consistent:

![image](https://user-images.githubusercontent.com/1061942/72592967-18fb5b80-3904-11ea-8ea2-630ed5b561b9.png)

Which means two types are fine here 👍